src/components/plotly/pyramid-bar/index.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { plotlyDefaultProps, plotlyPropTypes } from '../shared/constants'
+import CustomPlot from '../shared/custom-plot'
+import useTransformedData from '../shared/use-transformed-data'
+
+
+const PyramidBar = ({
+  data,
+  stacked,
+  showTicks,
+  showAxisTitles,
+  x,
+  y,
+  orientation,
+  percentage,
```

Comment:
Perhaps rename this as `showPercentage`. The prop name in config object in Widget Studio is named that way.